lib/web_ui/test/matchers.dart

```diff
@@ -496,3 +503,54 @@ final Matcher throwsAssertionError = throwsA(isAssertionError);
 ///  * [throwsAssertionError], to test if a function throws any [AssertionError].
 ///  * [isFlutterError], to test if any object is a [FlutterError].
 const Matcher isAssertionError = TypeMatcher<AssertionError>();
+
+/// Asserts that two [double]s are equal, within some tolerated error.
+///
+/// {@template flutter.flutter_test.moreOrLessEquals}
+/// Two values are considered equal if the difference between them is within
+/// [precisionErrorTolerance] of the larger one. This is an arbitrary value
+/// which can be adjusted using the `epsilon` argument. This matcher is intended
+/// to compare floating point numbers that are the result of different sequences
+/// of operations, such that they may have accumulated slightly different
+/// errors.
+/// {@endtemplate}
+///
+/// See also:
+///
+///  * [closeTo], which is identical except that the epsilon argument is
+///    required and not named.
+///  * [inInclusiveRange], which matches if the argument is in a specified
+///    range.
+///  * [rectMoreOrLessEquals] and [offsetMoreOrLessEquals], which do something
+///    similar but for [Rect]s and [Offset]s respectively.
+Matcher moreOrLessEquals(double value, { double epsilon = precisionErrorTolerance }) {
```

Comment:
This is not needed anymore?

Reply:
We still use this in many tests. Are you suggesting we use `within()` for everything?

Reply:
In flutter/engine/web_ui we don't need to introduce the matcher now, or do we? I only see it used in one test in this PR (and that could use `within` as well?)